### ISSUE TITLE
Issue with pictures in PF2 not working

### ISSUE THREAD

**Issue by SebastianH17** [ironmonk88/monks-enhanced-journal#245]
In PF2, when you create a new journal entry, it shows the "Description" and "Picture" options tabs, but the picture tab is no longer functional 
![image](https://user-images.githubusercontent.com/84858570/160494157-a7e00ef2-669f-4518-a6ee-49cade961d22.png)
In previous versions (I want to say 1.0.39 or 38 (I have a backup of that module version but can't see the version number right now) you could still click the picture to attach an artwork but this is no longer possible. 

If you convert the type of handout to "picture" you can attach a new artwork, but any previously linked artworks do no longer show. 
This is mostly an issue because a lot of PF2 people use pdftofoundry since that creates handouts for you. Though I did test creating a handout with art without this module active, and I am seeing the same issue.

The really weird thing is that the art is still kinda attached to the handout since it shows if you convert it to an organization, person, place, point of interest, quest, shop or loot... just not if you covert it to a Enhanced Journal picture handout. 

![image](https://user-images.githubusercontent.com/84858570/160494750-192baa59-9bf7-40a1-95cc-9228422b7e07.png)

My current workaround is to just open every handout, change the type to Picture and then re-attaching the artwork, but it would be neat if this could get fixed. 
It's a very neat module and I really love suggesting it to others^^

**Comment by ironmonk88** [comment]
Could you run Find the Culprit and see if this is a weird module interaction?
The picture tab is still working correctly for me.

**Comment by SebastianH17** [comment]
I tried or at least started, deactivating everything but this module on an entirely new world where no other modules are or were active. For some reason it worked on another world for a short moment but on the new one the tab  is not interactive - or rather I can only get to one interaction. 
The entire picture tab does not respond to left clicks (though I can go back to the description tab) but if I right click I see fractions of a option in the right that shows the art to players when clicked.

![image](https://user-images.githubusercontent.com/84858570/162612697-fd1cf272-28ae-4acc-8d5e-47f151299a81.png)

If I chose a piece of artwork before activating the module it shows a blue message 

![image](https://user-images.githubusercontent.com/84858570/162612756-fcacf857-0f49-4245-8b6f-56eb562fb2fd.png)

otherwise, it just gives an error message:

![image](https://user-images.githubusercontent.com/84858570/162612787-89c500c6-ce8b-4c81-af8a-f5a55adff863.png)

----------------------------

Sorry if this is not very helpful, all I can say is that I just used an old backup and V1.0.39 still works, right now in my current up-to-date world with all the other addons (with the module prevented from updating to a newer version). 

I really, appreciate this module, thank you for your hard work.
